database/entity/User.ts

```diff
@@ -1 +1 @@
-export { User } from './0037-drop_user_setting_table/User'
+export { User } from './0040-add_contribution_link_id_to_user/User'
```

Comment:
It has to be added a userId to the contributionLink Table, if such a relation will be necessary